packages/utils/src/types/actions.ts

```diff
@@ -292,6 +294,26 @@ export const CreateActionDetailSchema = z.object({
 export type CreateActionForm = z.infer<typeof CreateActionFormSchema>
 export type CreateActionDetail = z.infer<typeof CreateActionDetailSchema>
 
+/*
+COMPLETE
+*/
+export type CompleteActionParams = {
+  boostId?: string
+  actionType?: string
+}
+
+export const CompleteActionFormSchema = z.object({})
```

Comment:
The form schema is for the inputs we want to take in on manager, (at least this is how I understand it)

So we are going to want to have optional fields here for boostId and actionType